tm2/pkg/telemetry/config/config.go

```diff
@@ -2,32 +2,41 @@ package config
 
 import (
 	"errors"
+	"os"
 )
 
-var errEndpointNotSet = errors.New("telemetry exporter endpoint not set")
+var errEndpointNotSet = errors.New("telemetry endpoints not set")
 
 // Config is the configuration struct for the tm2 telemetry package
 type Config struct {
 	MetricsEnabled   bool   `toml:"enabled"`
 	MeterName        string `toml:"meter_name"`
+	PrometheusAddr   string `toml:"prometheus_laddr" comment:"expose prometheus endpoint on :26660, disabled if empty"`
 	ServiceName      string `toml:"service_name"`
+	ServiceInstance  string `toml:"service_instance"`
 	ExporterEndpoint string `toml:"exporter_endpoint" comment:"the endpoint to export metrics to, like a local OpenTelemetry collector"`
 }
 
 // DefaultTelemetryConfig is the default configuration used for the node
 func DefaultTelemetryConfig() *Config {
+	hostname, err := os.Hostname()
+	if err != nil {
+		hostname = "gno-node"
+	}
 	return &Config{
 		MetricsEnabled:   false,
+		PrometheusAddr:   ":26660",
```

Comment:
Can you please add a `PrometheusAddr` test to `config_test.go`?

Reply:
It's tested in 

```go
	t.Run("exporter endpoint not set", func(t *testing.T) {
		t.Parallel()

		c := DefaultTelemetryConfig()
		c.ExporterEndpoint = "" // empty
		c.PrometheusAddr = ""

		assert.ErrorIs(t, c.ValidateBasic(), errEndpointNotSet)
	})
```

Now that there is the 2 options, we set `errEndpointNotSet` only if both aren't set. So we can have otel without prom metrics and vise-versa